lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -3540,6 +3575,7 @@ class ConvertTorchToTosa : public ConvertTorchToTosaBase<ConvertTorchToTosa> {
     INSERT_ATENOP_PATTERN(AtenTanhOp);
     INSERT_ATENOP_PATTERN(AtenSigmoidOp);
     INSERT_ATENOP_PATTERN(AtenReluOp);
+    INSERT_ATENOP_PATTERN(AtenLeakyReluOp);
```

Comment:
https://github.com/ljfitz/torch-mlir/blob/main/python/torch_mlir_e2e_test/test_suite/elementwise.py#L348 should now pass for tosa-mlir. You should say that in https://github.com/ljfitz/torch-mlir/blob/main/e2e_testing/xfail_sets.py#L204.

We don't run this testing by default. Instructions for runnign them are in https://github.com/ljfitz/torch-mlir/blob/main/docs/development.md#running-execution-end-to-end-tests.

Reply:
Thanks! I never looked to deeply into their testing infrastructure.